kata.py

```diff
@@ -12,18 +19,34 @@ def Empty(string):
                 else:
                     answer = 0
                     for num in string[4:].split('\n'):
+                        Negative(num)
                         answer += int(num)
-                    return answer
+                    if len(negatives_list) == 0:
```

Comment:
I think this exact block of code gets repeated three times, which may suggest (not always, mind) it could be extracted as part of a refactoring step.